tools/pontoon/renaming_a_file.md

```diff
@@ -0,0 +1,15 @@
+# Renaming a localization file in a project without losing data
+
+In some cases, project owners will want to rename a file for clarity, or for technical reasons. If it’s only done on the code side, it will unfortunately require the file to be re-translated from scratch. Since this is a special case, we can work around this and rename the file in Pontoon while keeping existing data (translations, suggestions, attribution, history). The process is not complicated, but it has to be highly coordinated.
```

Comment:
```
If it’s only done on the code side, it will unfortunately require the file to be re-translated from scratch.
```

This is not correct: if you rename a file completely localized for locale X in the repository, Pontoon will read it, expose it as completely translated as it was before. What you're losing is history, attribution, pending suggestions.

the code side -> in the VCS repository